distributed-job-manager/src/main/kotlin/ru/fix/distributed/job/manager/flight/control/JobStateInspectorImpl.kt

```diff
@@ -0,0 +1,33 @@
+package ru.fix.distributed.job.manager.flight.control
+
+import org.apache.curator.framework.CuratorFramework
+import ru.fix.distributed.job.manager.ZkPathsManager
+import java.lang.IllegalArgumentException
+
+class JobStateInspectorImpl(
+    zkRootPaths: Set<String>,
```

Comment:
Why we treating DJM as if it could have several zkRootPaths simultaneously ? Seems like it can have only one and inspector should take only one path in ctor?

Reply:
Изначально, хотел сделать один инспектор для всех приложений. Однако с другой стороны будет понятнее при поиске ошибок, если будет одно приложение - один инспектор